### ISSUE TITLE
Counterfeiter doesn't add required imports if the package name doesn't match the folder

### ISSUE THREAD

**Issue by madamkiwi** [maxbrunsfeld/counterfeiter#55]
test/potato.go:

```go
package test

import (
	git "github.com/libgit2/git2go"
)

type Potato interface {
	Tomato(git.Oid)
}
```

after running `counterfeiter . Potato`

test/testfakes/fake_potato.go:

```go
// This file was generated by counterfeiter
package testfakes

import (
	"counterfeiter-bug-poc/test"
	"sync"
)

type FakePotato struct {
	TomatoStub        func(git2go.Oid)
	tomatoMutex       sync.RWMutex
	tomatoArgsForCall []struct {
		arg1 git2go.Oid
	}
	invocations      map[string][][]interface{}
	invocationsMutex sync.RWMutex
}

func (fake *FakePotato) Tomato(arg1 git2go.Oid) {
	fake.tomatoMutex.Lock()
	fake.tomatoArgsForCall = append(fake.tomatoArgsForCall, struct {
		arg1 git2go.Oid
	}{arg1})
	fake.recordInvocation("Tomato", []interface{}{arg1})
	fake.tomatoMutex.Unlock()
	if fake.TomatoStub != nil {
		fake.TomatoStub(arg1)
	}
}

func (fake *FakePotato) TomatoCallCount() int {
	fake.tomatoMutex.RLock()
	defer fake.tomatoMutex.RUnlock()
	return len(fake.tomatoArgsForCall)
}

func (fake *FakePotato) TomatoArgsForCall(i int) git2go.Oid {
	fake.tomatoMutex.RLock()
	defer fake.tomatoMutex.RUnlock()
	return fake.tomatoArgsForCall[i].arg1
}

func (fake *FakePotato) Invocations() map[string][][]interface{} {
	fake.invocationsMutex.RLock()
	defer fake.invocationsMutex.RUnlock()
	fake.tomatoMutex.RLock()
	defer fake.tomatoMutex.RUnlock()
	return fake.invocations
}

func (fake *FakePotato) recordInvocation(key string, args []interface{}) {
	fake.invocationsMutex.Lock()
	defer fake.invocationsMutex.Unlock()
	if fake.invocations == nil {
		fake.invocations = map[string][][]interface{}{}
	}
	if fake.invocations[key] == nil {
		fake.invocations[key] = [][]interface{}{}
	}
	fake.invocations[key] = append(fake.invocations[key], args)
}

var _ test.Potato = new(FakePotato)
```

**Comment by tjarratt** [comment]
Thanks for filing this issue @madamkiwi - I think this may even be a known defect with counterfeiter.

Are you interested in contributing to counterfeiter to fix this bug? I'd be happy to work with you to make this change.

**Comment by madamkiwi** [comment]
Yes, I’d love to!

**Comment by joefitzgerald** [comment]
Fixed in #61.
